### PR TITLE
Fix/envs

### PR DESCRIPTION
**Pull request by y-prudent**
**Small changes**:
- Change max line length to 90 for precommits
- Add seaborn and plotly dependancies to setup.py
- Add `-dev` suffix for dev envs in makefile
- Update README.md and index.md with reorganization of sections
- Remove deprecated "ignored_notebooks"

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -71,6 +71,8 @@
     "bump2version",
     "docsig",
     "no_implicit_optional",
+    "seaborn",
+    "plotly",
```
Comment (cofri) [inline setup.py:75]: I suppose seaborn and plotly should be in `requirements` variable instead of `dev_requirements`?

Comment (y-prudent) [reply setup.py:75]: +1

Comment (y-prudent) [reply setup.py]: +1

**Review by cofri — APPROVED** [review]

**Review by paulnovello — APPROVED** [review]
LGTM

**Pull request merged**